components/dashboard/src/App.tsx

```diff
@@ -38,6 +39,7 @@ const App: FC = () => {
         return <Login />;
     }
 
+    // TODO: Can we shift this into AppBlockingFlows as well? Does it matter if we wait until org query finishes to show this?
```

Comment:
Yes, we should shift it there and now it doesn't have dependencies other than the one it takes care of itself.